src/repository.js

```diff
@@ -25,6 +25,10 @@ export class Repository {
   getTransport() {
     if (this.transport === null) {
       this.transport = this.clientConfig.getEndpoint(this.getMeta().fetch('endpoint'));
+
+      if (!this.transport) {
+        throw new Error(`There is no endpoint '${this.getMeta().fetch('endpoint') || 'default'}' is registered.`);
```

Comment:
i actually figured that meanwhile. but also was too stupid to read your hint properly. :smile: